File: BackEnd/src/main/java/com/example/backend/model/Schedule.java
package com.example.backend.model;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
@Entity
public class Schedule {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Integer id;
    private String email;
    private String date;
    private String startTime;
    private String duration;
    private String patientName;

    private Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public String getStartTime() {
        return startTime;
    }

    public void setStartTime(String startTime) {
        this.startTime = startTime;
    }

    public String getDuration() {
        return duration;
    }

    public void setDuration(String duration) {
        this.duration = duration;
    }


    public String getpatientName() {
        return patientName;
    }

    public void setpatientName(String patientName) {
        this.patientName = patientName;
    }
}

File: ND_Telemedicine/lib/Models/schedule.dart
class Schedule {
  String? email;
  String? date;
  String? startTime;
  String? duration;

  Schedule({this.email, this.date, this.startTime, this.duration});

  // Schedule.fromJson(Map<String, dynamic> json) {
  //   email = json['email'];
  //   date = json['date'];
  //   startTime = json['startTime'];
  //   duration = json['duration'];
  // }

  factory Schedule.fromJson(Map<String, dynamic> json) {
    return Schedule(
    email : json['email'],
    date : json['date'],
    startTime : json['startTime'],
    duration : json['duration'],
    );
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['email'] = this.email;
    data['date'] = this.date;
    data['startTime'] = this.startTime;
    data['duration'] = this.duration;
    return data;
  }
}